CONTRIBUTING.md

```diff
@@ -137,6 +137,9 @@ The exact steps for an exemplary `0.2.0` release might look like this:
      description
  14. The `-dev` is stripped from the `0.2.0-dev` changelog entries on master
 
+All commits to master branch, which speculates with version number, like on
+steps `3` and `14`, should state that they don't represent a stable release.
```

Comment:
This doesn't really make sense to me, could you reword it a bit? Also I'm not sure if we explicitly need to point this out since this section is getting quite complicated? So we should omit everything that isn't absolutely necessary.

Reply:
Well, it seems like it's necessary, don't you think? Especially since you basically require it to be presented. I do agree that it's a bit too much at this point, and I had some troubles even phrasing it initially, so I'm not entirely sure how I'd rephrase it.

Reply:
I mean it's not really necessary, it's just a precaution. There's nothing really terrible that would happen, it's just to prevent issues from popping up before they even exist. If we don't document this and it's omitted, the worst that could happen is that people would figure it out on their own, right?

Reply:
that's fair.

You can close that PR, since I don't really like it myself.

Reply:
I'll close this for now, with the note that the entire section can probably be streamlined a bit in the future.